en_us/quests/hello_puppet.md

```diff
@@ -141,43 +120,37 @@ system that can access the master.
 
 <div class = "lvm-task-number"><p>Task 1:</p></div>
 
-To get started, use `ssh` to connect to `hello.puppet.vm` by running the command
-below. When prompted, enter the password `puppet`.
-
-    ssh learning@hello.puppet.vm
-
-Now that you're connected to this system, copy and run the following command to
+To get started, we will use the `bolt` tool that was introduced in the previous quest. Copy and run the following command to
```

Comment:
Great, I'll scan through and see how I can tighten that up. 👍

Reply:
I think that was the original text, so it's likely that all of the quests will need to be edited to remove excessive pronouns and use of `we` and `let's`. That will need to be a separate follow-on ticket for sure since it will be a multi-day effort.

Reply:
This is my edit to address the issue you raised: https://github.com/puppetlabs/puppet-quest-guide/pull/268/commits/4c8e15b42154d1693d969153f7b4aad67bfdfe67

How does that look @bridgetegan ?

Reply:
I think that is a great edit, @greglarkin! And since the pronoun use is legacy, I imagine that each time you edit the writing will tighten up. So, yeah, a big effort that may not be worth it on its own.